boards/arm/adafruit_trinket_m0/doc/adafruit_trinket_m0.rst

```diff
@@ -0,0 +1,149 @@
+.. _adafruit_trinket_m0:
+
+Adafruit Tinket M0
```

Comment:
s/Tinket/Trinket/

Reply:
Good spotting, fixed and ran a spell check on the file.